habitat/plan.sh

```diff
@@ -1,7 +1,7 @@
 pkg_name=hubs-discord-bot
 pkg_origin=mozillareality
 pkg_maintainer="Mozilla Mixed Reality <mixreality@mozilla.com>"
-pkg_version="0.0.2"
+pkg_version="0.0.5"
```

Comment:
You don't actually have to update this version with every push. The package is already versioned with a timestamp. I think we generally use this `pkg_version` when we need to signal some major backwards incompatibility issue, but even then it doesn't really apply to the discord bot.

Reply:
Should I revert or it's already too late?

Reply:
Ah, if you've already deployed this, it's too late. Not a big deal though, just a suggestion that you can avoid it next time.